docs/data/charts/scatter/ScatterCustomShape.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from 'react';
+import { ScatterChart, ScatterMarkerProps } from '@mui/x-charts/ScatterChart';
+import { ChartsLabelCustomMarkProps } from '@mui/x-charts/ChartsLabel';
+import { data } from './randomData';
+
+export default function ScatterCustomShape() {
+  return (
+    <ScatterChart
+      width={600}
+      height={300}
+      series={[
+        {
+          id: '1',
+          label: 'Series A',
+          data: data.map((v) => ({ x: v.x1, y: v.y1, id: v.id })),
+          markerSize: 1,
+          labelMarkType: StarLabelMark,
+        },
+        {
+          id: '2',
+          label: 'Series B',
+          data: data.map((v) => ({ x: v.x1, y: v.y2, id: v.id })),
+          markerSize: 1,
```

Comment:
I don't love it that `markerSize` is just a scale. I think it would be better if it were the width in pixels of a marker, for example.

Reply:
What do you mean by "scale"? It is the exact size of the radius in pixels. Naming it `size` was misleading though 😅

Reply:
Yeah, should we change it?

By scale I meant that it isn't the actual value, it's a scale applied to the default value.